charts/guard/templates/routing/httproute-subdomain.yaml

```
@@ -48,24 +48,36 @@ spec:
   {{- end }}
 
   rules:
-  # Determine the backend service to route to.
-  - backendRefs:
-    - name: {{ $.Values.global.serviceName }}
-      port: {{ $.Values.global.port }}
-
-    filters:
-    # Ensure the request header "target-service-id" is set to the authoritative service ID.
-    - type: RequestHeaderModifier
-      requestHeaderModifier:
-        set:
-        - name: "target-service-id"
-          value: "{{ .serviceId }}"
-
-    matches:
-    # Match any path prefix that starts with "/v1"
-    - path:
-        type: PathPrefix
-        value: /v1
+    # -- CONTROLLED TRAFFIC SHIFTING --
+    # Define a default dict for the traffic splitting configuration.
+    # This is to prevent nil pointer errors when the traffic splitting
+    # configuration is not set for a service.
```

This will allow helm-docs to use the whole comment as value description.

```suggestion
    #-- Define a default dict for the traffic splitting configuration.
    #-- This is to prevent nil pointer errors when the traffic splitting
    #-- configuration is not set for a service.
```

---

> This will allow helm-docs to use the whole comment as value description.

Does this work for template files? I thought it only picked up `values.yaml`? This isn't a values field description.